shared/common-adapters/popup-dialog.desktop.js

```diff
@@ -64,6 +64,7 @@ const coverStyle = {
   paddingRight: globalMargins.large,
   paddingTop: globalMargins.small,
   paddingBottom: globalMargins.small,
+  zIndex: 1, // Put the popup on top of any sticky section headers.
```

Comment:
My mental model for how zIndex works isn't great, so if there's a clearer way to word this let me know!